guides/json.md

```diff
@@ -0,0 +1,69 @@
+- [Making Raw JSON REST Requests](#making-raw-json-rest-requests)
+  - [GET](#get)
+  - [PUT](#put)
+  - [POST](#post)
+  - [DELETE](#delete)
+
+# Making Raw JSON REST Requests
+OpenSearch exposes a REST API that you can use to interact with OpenSearch. The OpenSearch .NET client provides a low-level API that allows you to send raw JSON requests to OpenSearch. This API is useful if you want to use a feature that is not yet supported by the OpenSearch .NET client, but it supported by the OpenSearch REST API.
+
+## GET
+The following example returns the server version information via `GET /`.
+
+```csharp
+var versionResponse = await client.LowLevel.DoRequestAsync<DynamicResponse>(HttpMethod.GET, "/", CancellationToken.None);
+Console.WriteLine(versionResponse.Body["version"]["distribution"].ToString(), versionResponse.Body["version"]["number"].ToString()); // Distribution & Version number
+```
+
+# PUT
+The following example creates an index.
+
+```csharp
+var indexBody = new 
+{
+    settings = new 
+    {
+        index = new 
+        {
+            number_of_shards = 4
+        }
+    }
+};
```

Comment:
Is there a JSON/string version of this? Not sure what's best for this example, maybe both?